dcos/cosmos.py

```diff
@@ -227,7 +210,7 @@ def _get_http_method(self, endpoint):
         """
         return self._endpoint_data.get(endpoint).get('http_method')
 
-    def _get_header(self, endpoint, version, headers=None):
+    def _get_header(self, endpoint, versions, headers=None):
```

Comment:
Should we just change this to `_get_header(self, endpoint, headers=None)`? Don't we always want to send all of the version that we support?